### PR TITLE
Add header changes for SPV_EXT_shader_atomic_float_min_max

### PR DESCRIPTION
**Pull request by jekstrand**
See also https://github.com/KhronosGroup/SPIRV-Registry/pull/92

### CODE REVIEW

File: include/spirv/unified1/spirv.core.grammar.json [hunk 1]
```diff
@@ -7099,6 +7099,38 @@
...
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",             "name" : "'Pointer'" },
+        { "kind" : "IdScope",           "name" : "'Memory'" },
+        { "kind" : "IdMemorySemantics", "name" : "'Semantics'" },
+        { "kind" : "IdRef",             "name" : "'Value'" }
+      ],
+      "capabilities" : [ "AtomicFloat16MinMaxEXT", "AtomicFloat32MinMaxEXT", "AtomicFloat64MinMaxEXT" ],
+      "extensions" : [ "SPV_EXT_shader_atomic_float_min_max" ],
```
Comment (dneto0) [inline include/spirv/unified1/spirv.core.grammar.json:7116]: I suggest not having this "extensions" list for non-capabilities.  Guard this token by capability only, and guard the capability by the extension.

File: include/spirv/unified1/spirv.core.grammar.json [hunk 2]
```diff
@@ -7099,6 +7099,38 @@
       "capabilities" : [ "AtomicFloat32AddEXT", "AtomicFloat64AddEXT" ],
       "extensions" : [ "SPV_EXT_shader_atomic_float_add" ],
       "version" : "None"
+    },
+    {
+      "opname" : "OpAtomicFMinEXT",
+      "class"  : "Atomic",
+      "opcode" : 5614,
```
Comment (dneto0) [inline include/spirv/unified1/spirv.core.grammar.json:7106]: Please move this up so the instructions appear in numeric order.

**Review by dneto0 — CHANGES_REQUESTED** [review]
I have verified that this works with SPIRV-Tools at the top of tree.

File: include/spirv/unified1/spirv.core.grammar.json (hunk 1 above)
Comment (jekstrand) [reply include/spirv/unified1/spirv.core.grammar.json:7116]: Done.

File: include/spirv/unified1/spirv.core.grammar.json (hunk 2 above)
Comment (jekstrand) [reply include/spirv/unified1/spirv.core.grammar.json:7106]: Done.

**Comment by jekstrand** [comment]
I also just fixed an issue where we were accidentally depending on the **Shader** capability.  This is intended to work with OpenCL as well as Vulkan.

**Review by dneto0 — APPROVED** [review]
Looks good to me now too.
I verified this builds correctly with SPIRV-Tools top of tree.

**Comment by jekstrand** [comment]
I just rebased on master and resolved the merge conflicts.

**Comment by jekstrand** [comment]
@johnkslang this has been sitting for 2 weeks.  Any reason it's not gone in yet?

**Pull request merged**